test/object_store/run

```diff
@@ -51,6 +51,13 @@ def teardown(pid):
     log = run.abort_run_with_dir(pid, test_tempdir)
     shutil.copyfileobj(log, sys.stdout.buffer)
 
+
+def format_map(m):
```

Comment:
This looks generic enough to be placed outside of the narrow object_store/run

Reply:
moved into `test/cql-pytest/util.py`.